api/lib/opentelemetry/trace/tracer.rb

```diff
@@ -31,22 +31,22 @@ def current_span
       #
       # With this helper:
       # OpenTelemetry.tracer.in_span('do-the-thing') do ... end
-      def in_span(name, sampler: nil, links: nil, recording_events: nil, kind: nil)
-        span = start_span(name, sampler: sampler, links: links, recording_events: recording_events, kind: kind)
+      def in_span(name, attributes: nil, links: nil, events: nil, resource: nil, span_id: nil, start_timestamp: nil)
```

Comment:
Should we document the parameters to the API methods in Tracer? Or should we handle this separately?

Reply:
We should document them all.